content/02.introduction.md

```diff
@@ -9,12 +9,20 @@ The low survival rates for some pediatric tumors are clearly multifactorial, exp
 
-Although there has been significant progress in recent years to elucidate the landscape of somatic variation responsible for pediatric brain tumor formation and progression, translation of therapeutic agents to phase II or III clinical trials and subsequent FDA approvals have not kept pace. 
-Within the last 20 years, the FDA has approved only five drugs for the treatment of pediatric brain tumors: mTOR inhibitor, everolimus, for subependymal giant cell astrocytoma [@doi:10.1097/01.COT.0000421359.88140.a8; @url:https://www.fda.gov/drugs/resources-information-approved-drugs/fda-approves-everolimus-tuberous-sclerosis-complex-associated-partial-onset-seizures]; anti-PD-1 immunotherapy, pembrolizumab, for microsatellite instability–high or mismatch repair–deficient tumors [@url:https://www.fda.gov/drugs/drug-approvals-and-databases/fda-approves-pembrolizumab-adults-and-children-tmb-h-solid-tumors]; NTRK inhibitors larotrectinib [@url:https://www.fda.gov/drugs/fda-approves-larotrectinib-solid-tumors-ntrk-gene-fusions] and entrectinib [@url:https://www.fda.gov/drugs/resources-information-approved-drugs/fda-approves-entrectinib-ntrk-solid-tumors-and-ros-1-nsclc] for tumors with an NTRK 1/2/3 gene fusions; MEK1/2 inhibitor, selumetinib, for neurofibromatosis type 1 (NF1) and symptomatic, inoperable plexiform neurofibromas [@url:https://www.fda.gov/drugs/resources-information-approved-drugs/fda-approves-selumetinib-neurofibromatosis-type-1-symptomatic-inoperable-plexiform-neurofibromas]. 
-This is, in part, due to pharmaceutical company priorities and/or concerns regarding toxicity leading, making it challenging for researchers to obtain to new therapeutic agents for pediatric clinical trials.
-Critically, as of August 18, 2020, an amendment to the Pediatric Research Equity Act called the "Research to Accelerate Cures and Equity (RACE) for Children Act" mandates that all new adult oncology drugs also be tested in children when the molecular targets are relevant to a particular childhood cancer. 
-This regulatory change is poised to accelerate identification of novel and effective therapeutic for pediatric diseases that have otherwise been overlooked.
+To leverage diverse scientific and analytical expertise to analyze the PBTA data, we created an open science model and incorporated features, such as analytical code review [@doi:10.1093/aje/kwab092; @doi:10.7287/peerj.preprints.3210v1] and continuous integration to test data and code [@doi:10.1038/nbt.3780; @doi:10.7287/peerj.preprints.3210v1], to improve reproducibility throughout the life cycle of our project, termed OpenPBTA.
+We anticipated that a model of open collaboration would enhance the value of our effort to the pediatric brain tumor research community. 
+Openly sharing data and code in real time allows others to build upon our work more rapidly, and publications that include data and code sharing may have a greater impact [@doi:10.1371/journal.pone.0000308; @doi:10.1371/journal.pcbi.1008867].
```

Comment:
```suggestion
Openly sharing data and code in real time allows others to build upon our work more rapidly, and publications that include data and code sharing may have a greater impact [@doi:10.1371/journal.pone.0000308; @doi:10.1371/journal.pcbi.1008867].
```
 
Is this a _may_ or it it _more likely to have_ a greater impact. I think using something along the lines of the latter is a bit more convincing, but open to wording here.

Reply:
It depends on what we mean by "greater impact" I suppose. If someone makes their data publicly available, they are more likely to get cited (per `@doi:10.1371/journal.pone.0000308`). Code sharing means something is more transparent and can be built upon (per the editorial `@@doi:10.1371/journal.pcbi.1008867`). So that's to say – there's evidence that data sharing results in more citations and it makes intuitive sense that sharing code means it can be used for other applications, but I don't think that's been studied.

For what it's worth, we're going to tell people that we made an impact on other projects in the discussion!